resources/migrations/001_update_migrations.yaml

```diff
@@ -358,7 +358,7 @@ databaseChangeLog:
                   type: char(21)
                   remarks: Random NanoID tag for unique identity.
                   constraints:
-                    nullable: true
+                    nullable: false
```

Comment:
are you changing old migration here?

Reply:
oh yea I shouldn't do that, will revert

Reply:
a41e9b4946

Reply:
it was accidentally changed on the feature branch https://github.com/metabase/metabase/pull/47857/files#diff-e6ec4fc8084d80ae2ebedde98c98f358ec7ac526de5c339ebcb454cbda62b221